tailcfg/tailcfg.go

```diff
@@ -926,6 +926,14 @@ type PingRequest struct {
 	// Log is whether to log about this ping in the success case.
 	// For failure cases, the client will log regardless.
 	Log bool `json:",omitempty"`
+
+	// Types is the types of ping that is initiated. Can be TSMP, ICMP or disco.
+	// Types will be comma separated, such as TSMP, disco.
```

Comment:
such "TSMP,disco" I assume you mean? I don't think we want to permit spaces in there.